packages/framework/src/Command/ElFinderInstallerCommand.php

```diff
@@ -26,20 +26,22 @@ final class ElFinderInstallerCommand extends Command
 
     private const ELFINDER_IMG_DIR = 'studio-42/elfinder/img';
 
+    /**
+     * @var string
+     * @phpcsSuppress SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingNativeTypeHint
+     * @phpcsSuppress SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnlySniff.ReferenceViaFullyQualifiedName
+     */
     protected static $defaultName = 'elfinder:install';
 
-    protected $fileSystem;
-
-    protected $parameterBag;
+    protected Filesystem $fileSystem;
 
     /**
      * @param \Symfony\Component\Filesystem\Filesystem $filesystem
      * @param \Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface $parameterBag
      */
-    public function __construct(Filesystem $filesystem, ParameterBagInterface $parameterBag)
+    public function __construct(Filesystem $filesystem, protected readonly ParameterBagInterface $parameterBag)
```

Comment:
I wonder, why the Filesystem is not converted to cpp

Reply:
Updated manually